src/pages/applications/Applications.jsx

```diff
@@ -526,7 +532,7 @@ const Applications = () => {
                         </div>
                         <div className="application-gender">
                           <p>Gender</p>
-                          <span>-</span>
+                          <span>{application?.gender?.charAt(0).toUpperCase()}{application?.gender?.slice(1).toLowerCase()}</span>
```

Comment:
make this reusable function, because i have seen lot places uses the same logic.